migration/idempotent/007-tracing-tags.sql

```diff
@@ -38,17 +40,18 @@ LANGUAGE SQL STABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION _ps_trace.eval_tags_by_key(ps_trace.tag_k) TO prom_reader;
 
 CREATE OR REPLACE FUNCTION _ps_trace.has_tag(_tag_map ps_trace.tag_map, _key ps_trace.tag_k)
-RETURNS boolean
+    RETURNS boolean
+    -- Note: no explicit `SET SCHEMA` because we want this function to be inlined
 AS $func$
-    SELECT _tag_map @> ANY(_ps_trace.eval_tags_by_key(_key))
+    SELECT _tag_map OPERATOR(pg_catalog.@>) ANY(_ps_trace.eval_tags_by_key(_key))
 $func$
 LANGUAGE SQL STABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION _ps_trace.has_tag(ps_trace.tag_map, ps_trace.tag_k) TO prom_reader;
 COMMENT ON FUNCTION _ps_trace.has_tag IS $$This function supports the #? operator.$$;
 
 DO $do$
 BEGIN
-    CREATE OPERATOR ps_trace.#? (
+    CREATE OPERATOR ps_trace.# (
```

Comment:
why was the name of the operator changed?

Reply:
oh man, that was an error. good catch.